services/src/main/java/org/keycloak/services/resources/account/AccountCredentialResource.java

```diff
@@ -341,116 +341,4 @@ public void setLabel(final @PathParam("credentialId") String credentialId, Strin
 //        session.userCredentialManager().moveCredentialTo(realm, user, credentialId, newPreviousCredentialId);
 //    }
 
```

Comment:
We have some unused imports which are safe to remove like:

```
import org.keycloak.credential.PasswordCredentialProvider;
import org.keycloak.credential.PasswordCredentialProviderFactory;
import org.keycloak.events.EventType;
import javax.ws.rs.POST;
```

We can also remove:
```
private final EventBuilder event;
```

Reply:
Good catch. Thanks, I removed that.